website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -362,3 +635,206 @@ static SnowflakeIO.CsvMapper<GenericRecord> getCsvMapper() {
            };
 }
 {{< /highlight >}}
+## Cross Language
```

Comment:
I wonder if this should be called 'Using SnowflakeIO in Python SDK' - what do you think? Users may not care if it's cross-lang, but rather that it works in their language of choice. Thoughts?

Reply:
+1

It's worth having a note somewhere like "SnowflakeIO in Python can only be used on runners that support cross-language transforms". I'm wary of listing specific runners here since it will get out of date. It seems we should have a central location that lists runners that support cross-language, seems like a good thing to have on the compatibility matrix.

Reply:
I changed to `Using SnowflakeIO in Python SDK'` and updated a little intro. 

About the second maybe lets create an issue for this?

Reply:
@pabloem @TheNeuralBit WDYT?